### DISCUSSION TITLE
Welcome to wg-manager Discussions!

### DISCUSSION THREAD

**Discussion by perara** [perara/wg-manager#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidnetten** [comment]
I may be unskilled, but I'm having a hard time to give this a spin. I'm trying to deploy the docker image on a Amazon Linux 2. I had it working, but after a lot of errors during installation, the GUI was not able to generate keypairs.

**Comment by skaylerj** [comment]
Help please i cant install this proggram
